### ISSUE TITLE
Duplicate HPO term

### ISSUE THREAD

**Issue by julienN75** [obophenotype/human-phenotype-ontology#7469]
**HPO term**
HP:5000006
HP:5000041

**Bug description**
HP:5000006 and HP:5000041 describe the same autoantibody positivity. Can you please merge the terms ?

**Comment by julienN75** [comment]
@MaudDLVT for follow up

**Comment by pnrobinson** [comment]
These are
[Anti-CV2/CRMP5 antibody HP:5000006](https://hpo.jax.org/app/browse/term/HP:5000006)
and
[Anti-CRMP5 antibody positivity HP:5000041](https://hpo.jax.org/app/browse/term/HP:5000041)
@julienN75  which label should be the preferred label -- Anti-CV2/CRMP5 antibody or Anti-CRMP5 antibody ?

**Comment by pnrobinson** [comment]
OK I have merged the terms.
The final term is

Anti-CV2/CRMP5 antibody positivity
HP:5000006
